imports/plugins/core/versions/server/migrations/65_update_navigation_tree.js

```diff
@@ -0,0 +1,43 @@
+import { Migrations } from "meteor/percolate:migrations";
+import Logger from "@reactioncommerce/logger";
+import rawCollections from "/imports/collections/rawCollections";
+import { setDefaultsForNavigationTreeItems } from "../util/convert65";
+
+const { NavigationTrees } = rawCollections;
+
+Migrations.add({
+  version: 65,
+  async up() {
+    // Find all navigation trees
+    const navigationTrees = await NavigationTrees.find({}).toArray();
+
+    // Update default values for each tree
+    Promise.all(navigationTrees.map((tree) => {
```

Comment:
Since this runs in a Meteor fibers context, you should have `Promise.await` around `Promise.all`. Otherwise it will think the migration is done before it's actually done.

```js
Promise.await(Promise.all(navigationTrees.map((tree) => {
```

Reply:
And then I think you'd use a normal try/catch around that if you want to catch and log the error.